docs/en/02_Developer_Guides/19_GraphQL/04_security_and_best_practices/02_cors.md

```diff
@@ -29,100 +29,94 @@ Once you have enabled CORS you can then control four new headers in the HTTP Res
 
 1. **Access-Control-Allow-Origin.**
 
- This lets you define which domains are allowed to access your GraphQL API. There are
- 4 options:
+   This lets you define which domains are allowed to access your GraphQL API. There are
+   4 options:
 
- * **Blank**:
- Deny all domains (except localhost)
+   * **Blank**: Deny all domains (except localhost)
 
- ```yaml
- Allow-Origin:
- ```
+     ```yaml
```

Comment:
Why the indent?

Reply:
Check out the rendered file before and after the indentation - before the numbered list doesn't render correctly, but after it does.